pysatSeasons/plot.py

```diff
@@ -1,52 +1,62 @@
-from __future__ import print_function
-from __future__ import absolute_import
+"""Support scatterplot production over seasons of interest."""
 
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D  # noqa: F401
```

Comment:
Why is this being imported if it isn't used?

Reply:
It is being used it just isn't obvious from the code. One of the axes generated below uses the '3d' projection, which requires Axes3D to be around.

Reply:
So... it's not obvious to the user or to flake8?  Maybe expand the note to explain why it's needed.